app/assets/javascripts/host_edit.js

```diff
@@ -283,6 +283,18 @@ function update_form(element, options) {
   })
 }
 
+function serializeForm() {
+  return $($('form')[0].elements).filter(isExplicit).serialize()
+}
+
+function isExplicit(index, element) {
```

Comment:
* What is this doing exactly? Maybe a comment would be necessary here.
* What is element, form elements? If so, why is it calling nextSibling on each ot them?
* Is there a way to avoid that chain of if checks?